### PR TITLE
COP-1955: Add tab component

### PR DESCRIPTION
**Pull request by JimmySunDance**
## Description
This branch adds the Tab component to the IDE controller front end. As part of this a place holder MRZ table has been added for the purposes of demonstrations to stake holders whilst the component is in development. 
There are also changes to how the govuk style library is implemented, to remove conflicts. 

## Developer Checklist

\* Required

- [x] Up-to-date with master branch? \*

- [ ] Does it have tests?

- [x] Pull request URL linked to JIRA ticket? \*

- [ ] All reviewer comments resolved/answered? \*

### CODE REVIEW

File: packages/react/src/Components/ImagePanel.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// Local imports 
+import ChipImage from './ChipImage';
+import LiveImage from './LiveImage';
+import ScanImage from './ScanImage';
+import PhotoHeaders from './PhotoHeaders'
+
+
+const ImagePanel = ({ isActive }) => {
+
+  console.log(isActive)
```
Comment (p3dr0migue1) [inline packages/react/src/Components/ImagePanel.jsx:13]: please remove console log :grimacing:

File: packages/react/src/Components/ImagePanel.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+// Global imports
+import React from 'react';
+
+// Local imports 
+import ChipImage from './ChipImage';
+import LiveImage from './LiveImage';
+import ScanImage from './ScanImage';
+import PhotoHeaders from './PhotoHeaders'
```
Comment (p3dr0migue1) [inline packages/react/src/Components/ImagePanel.jsx:8]: missing semi-colon ;) you probably need to run linter too

File: packages/react/src/Components/MRZPanel.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// Local imports
+import MRZTable from './MRZTable';
+
+const MRZPanel = ({ isActive }) => {
+
+  return (
+    <div 
+        className={`
+          govuk-tabs__panel
+          ${isActive ? '' : 'govuk-tabs__panel--hidden'}`}
```
Comment (p3dr0migue1) [inline packages/react/src/Components/MRZPanel.jsx:13]: you could probably fit this code in one line, its perfectly valid, but probably easier to read if in one line, unless linter says the opposite

File: packages/react/src/Components/ImagePanel.jsx (hunk 2 above)
Comment (JimmySunDance) [reply packages/react/src/Components/ImagePanel.jsx:8]: linter is currently not working.

**Review by rjweeks — APPROVED** [review]

**Pull request merged**